Remove unessessary aligned attribute which breaks Clang

This patch fixes errors when compiling the Linux kernel with Clang
This patch probably can't be sent upstream

Signed-off-by: Mark Charlebois <charlebm@gmail.com>
---
 shash.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

Index: linux/crypto/shash.c
===================================================================
--- linux.orig/crypto/shash.c	2012-09-06 14:07:04.858905273 -0400
+++ linux/crypto/shash.c	2012-09-06 14:07:48.686354288 -0400
@@ -67,7 +67,8 @@
 static inline unsigned int shash_align_buffer_size(unsigned len,
 						   unsigned long mask)
 {
-	return len + (mask & ~(__alignof__(u8 __attribute__ ((aligned))) - 1));
+	// FIXME (aligned not supported) - return len + (mask & ~(__alignof__(u8 __attribute__ ((aligned))) - 1));
+	return len + (mask & ~(__alignof__(u8) - 1));
 }
 
 static int shash_update_unaligned(struct shash_desc *desc, const u8 *data,
